images/update_kcp_stable/bin/gitlab_open_mr.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+declare -r SOURCE_BRANCH="robot/auto_update"
```

Comment:
There might be other robots that submits update for other things (e.g. kubeconfigs to add/remove clusters, change secrets, etc).
Therefore I would make the name a bit more descriptive, such as `robot/pipeline-service-update`.